printer.py

```diff
@@ -788,6 +771,25 @@ def _main():
             pass
         info(i18n('connecting'))
         printer.connect(devices[choice].name, devices[choice].address)
+
+    # Prepare image / text
+    print(f"printer: {printer.model}")
```

Comment:
Is this a debug message or intended output?
Consider removing it or localizing it :)

Reply:
Oops, sorry about that!